CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-edv-storage ChangeLog
 
+## 4.1.2 - 2020-07-09
```

Comment:
Needs to be a major bump due to the breaking change.

```suggestion
## 5.0.0 - 2020-07-xx
```

Reply:
Fixed.